python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -374,15 +374,45 @@ def update_trip_stop_counts(db_manager: DatabaseManager) -> None:
 
 def update_static_trip_id_guess_exact(db_manager: DatabaseManager) -> None:
     """
-    Update static_trip_id_guess with exact matches between rt and static trips
+    Update
+     - static_trip_id_guess
+     - static_stop_count
+     - static_start_time
+
+    for trips with exact matches to static trips
     """
+    static_stop_sub = (
+        sa.select(
+            StaticStopTimes.static_version_key,
+            StaticStopTimes.trip_id,
+            sa.func.count(StaticStopTimes.stop_sequence).label(
+                "static_stop_count"
```

Comment:
this is how many stops a trip is scheduled to hit?

Reply:
Correct